src/Microsoft.TemplateEngine.Edge/Settings/TemplatePackageManager.cs

```diff
@@ -170,8 +170,8 @@ public async Task<IReadOnlyList<ITemplateInfo>> GetTemplatesAsync(CancellationTo
         /// <param name="cancellationToken">A cancellation token to cancel the asynchronous operation.</param>
         /// <returns>The filtered list of templates with match information.</returns>
         /// <example>
-        /// <c>GetTemplatesAsync(WellKnownSearchFilters.MatchesAllCriteria, new [] { WellKnownSearchFilters.NameFilter("myname") }</c> - returns the templates which name or short name contains "myname". <br/>
-        /// <c>GetTemplatesAsync(TemplateListFilter.MatchesAtLeastOneCriteria, new [] { WellKnownSearchFilters.NameFilter("myname"), WellKnownSearchFilters.NameFilter("othername") })</c> - returns the templates which name or short name contains "myname" or "othername".<br/>
+        /// <c>GetTemplatesAsync(WellKnownSearchFilters.MatchesAllCriteria, new [] { WellKnownSearchFilters.NameFilter("name") }</c> - returns the templates which name or short name contains "name". <br/>
+        /// <c>GetTemplatesAsync(TemplateListFilter.MatchesAtLeastOneCriteria, new [] { WellKnownSearchFilters.NameFilter("name"), WellKnownSearchFilters.NameFilter("other") })</c> - returns the templates which name or short name contains "name" or "other".<br/>
```

Comment:
Since this is example text (which may/may not get published as reference docs), we probably shouldn't change it since the words it is changing are example names.